### ISSUE TITLE
Failed to import version.json file

### ISSUE THREAD

**Issue by kbolander** [scanoss/minr#13]
I've been trying to run through the test example in the [README ](https://github.com/scanoss/minr) section of the documentation, but whenever I get to the step of importing the data to the LDB (minr -i mined/), I'm getting an error saying "Failed to import version.json file. This file must be present and must has a valid format to continue." 

Looking at the help text, I see this information:
![image](https://user-images.githubusercontent.com/96793624/150861959-d8744af7-3667-42e1-b8ea-d21a1f397181.png)

But, even if I do create a version.json file, I can't seem to get past this error. I feel like there is something very simple that I'm missing to get this to work, what is it??

**Comment by mscasso-scanoss** [comment]
Dear @kbolander I'm deeply sorry for you troubles. We have added this change recently and We forgot to update the documentation properly. 

You are right, the version.json file is needed, and should be in the root of the mined path, typically  ./mined/version.json.
Attached you will find an example of this file (remove the ".txt" from the extension please). 

Please let me know if your problem was solved. I'm going to update the readme  pretty soon.
[version.json.txt](https://github.com/scanoss/minr/files/7929985/version.json.txt)

**Comment by kbolander** [comment]
Thanks @mscasso-scanoss, that worked! It might also be worth mentioning when you update the instructions that the minr -i mined/ call should be run above the mined/ directory. I first tried to run it within the mined/ directory (since I was there by default after updating my version.json file), and still received the error about the missing version.json file. It wasn't until I moved up above the mined/ directory that it was able to find the version.json properly and execute. 

Thanks again for your quick reply!
